weave/mongo_loader.py

```diff
@@ -179,3 +178,39 @@ def load_mongo(self, uuids, **kwargs):
         self.load_mongo_metadata(uuids, collection=metadata_collection)
         self.load_mongo_manifest(uuids, collection=manifest_collection)
         self.load_mongo_supplement(uuids, collection=supplement_collection)
+
+
+    @staticmethod
```

Comment:
Does this have to be a static method? All others are not, and it's just duplicating the code to connect to the correct db

Reply:
I made it into an instance method